pkg/cmd/server/origin/master.go

```diff
@@ -128,6 +128,12 @@ type MasterConfig struct {
 	DeployerOSClientConfig kclient.Config
 }
 
+// SupportInstaller installs support components into this server
+type SupportInstaller interface {
```

Comment:
Why a separate interface?  Why not make APIInstaller generic and have both API and support use the same one?  Is there some later divergence expected?